apm-agent-plugins/apm-jdbc-plugin/src/main/java/co/elastic/apm/agent/jdbc/signature/SignatureParser.java

```diff
@@ -57,9 +60,10 @@
     private final static ConcurrentMap<String, String> signatureCache = new ConcurrentHashMap<String, String>(DISABLE_CACHE_THRESHOLD, 0.5f, Runtime.getRuntime().availableProcessors());
 
     private final Scanner scanner = new Scanner();
+    private final StringBuilder dbLink = new StringBuilder();
 
     public void querySignature(String query, StringBuilder signature, boolean preparedStatement) {
-
+		dbLink.setLength(0);
```

Comment:
This probably should be another out parameter, like `signature`. Perhaps another method overload.